Android/app/src/main/java/io/github/project_travel_mate/login/LoginPresenter.java

```diff
@@ -71,12 +72,18 @@ public void onResponse(Call call, final Response response) throws IOException {
                     @Override
                     public void run() {
                         try {
+                            /* removed in order to direct t login on signup
                             JSONObject ob = new JSONObject(res);
                             String id = ob.getString("user_id");
                             view.rememberUserInfo(id, name, num);
                             view.startMainActivity();
+                            */
+
+                            view.openLogin();
+                            view.setLoginNumber(num);
+                            view.showMessage("signup succeeded! please login");
                             view.dismissLoadingDialog();
-                        } catch (JSONException e) {
+                        } catch (Exception e) {
```

Comment:
IMHO, catching a more specific exception is a better practice than catching a general one. Any particular reason for the change in this method?

Reply:
the JSONException was erased due to the changed of methodology 
so the java vm doesn't accept this specific exception
so instead of erasing the catch statment I changed it to a general catch

Reply:
makes sense! Overlooked the commented code.